acceptancetests/assess

```diff
@@ -263,14 +381,151 @@ def main():
     if args.keep_env:
         testrun_argv += ["--keep-env"]
     if args.region:
-        testrun_argv += ["--region=" + args.region]
+        testrun_argv += ["--region", args.region]
     if args.to:
-        testrun_argv.extend(["--to", args.to])
+        testrun_argv += ["--to", args.to]
     if args.agent_url:
-        testrun_argv.extend(["--agent-url", args.agent_url])
+        testrun_argv += ["--agent-url", args.agent_url]
     if args.agent_stream:
-        testrun_argv.extend(["--agent-stream", args.agent_stream])
-
+        testrun_argv += ["--agent-stream", args.agent_stream]
+
+    assess = args.assess
```

Comment:
Can we add a big comment that the below test specific behaviour is only for tests which use bespoke args for that test; tests which just use generic args are not listed etc etc

Reply:
Sure, done.